src/test/java/org/folio/inventory/dataimport/handlers/actions/CreateMarcHoldingsEventHandlerTest.java

```diff
@@ -365,7 +365,7 @@ public void shouldProcessEventIfInstanceFoundButTotalRecordsIsNotEqualOne() thro
 
     CompletableFuture<DataImportEventPayload> future = createMarcHoldingsEventHandler.handle(dataImportEventPayload);
 
-    DataImportEventPayload actualDataImportEventPayload = future.get(5, TimeUnit.SECONDS);
+    DataImportEventPayload actualDataImportEventPayload = future.get(10, TimeUnit.SECONDS);
```

Comment:
Why was this timeout changed? Was it due to intermittent failures in these tests?

Reply:
Yes.  This is noted in the PR comments.

Reply:
Apologies for not checking the notes